src/index.html

```diff
@@ -22,6 +22,383 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo__image"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <!-- is-active class -->
+          <li class="nav__item">
+            <a
+              href="#samsung"
```

Comment:
There's a typo in the `href` attribute value. It should be `#samsung` instead of `#samsund`.